### PR TITLE
docs: add Namron Zigbee FOH Green Bryter K4 - 4512727 to White-Label list

### PR DESCRIPTION
**Pull request by ruant**
Added this fine to z2m using the pairing steps provided.

### CODE REVIEW

File: docs/devices/PTM_215Z.md [hunk 1]
```diff
@@ -78,3 +78,6 @@ It's not possible to read (`/get`) or write (`/set`) this value.
 The minimal value is `0` and the maximum value is `255`.
 The unit of this value is `lqi`.
 
+## Home Assistant
```
Comment (Koenkk) [inline docs/devices/PTM_215Z.md:81]: Please move above `<!-- Notes END: Do not edit below this line -->`

File: docs/devices/PTM_215Z.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ pageClass: device-page
 | Description | Pushbutton transmitter module |
 | Exposes | action, linkquality |
 | Picture | ![EnOcean PTM 215Z](https://www.zigbee2mqtt.io/images/devices/PTM-215Z.png) |
-| White-label | Niko 91004, NodOn CWS-4-1-01_HUE, Vimar 03906, Sunricher SR-ZGP2801K4-FOH-E, LED-Trading 9125, Feller 4120.2.S.FMI.61 |
+| White-label | Niko 91004, NodOn CWS-4-1-01_HUE, Vimar 03906, Sunricher SR-ZGP2801K4-FOH-E, LED-Trading 9125, Feller 4120.2.S.FMI.61, Namron ZigBee FOH Green Bryter K4 (4512727) |
```
Comment (Koenkk) [inline docs/devices/PTM_215Z.md:23]: This is generated from https://github.com/Koenkk/zigbee-herdsman-converters/blob/cc8bff02f706007f990126310fee13f1e64781f4/src/devices/enocean.ts#L16, could you update that also?

Comment (ruant) [reply docs/devices/PTM_215Z.md:23]: I've added a PR for the change If it's generated from the file you, do you want me to remove it from here?

Comment (Koenkk) [reply docs/devices/PTM_215Z.md:23]: It's fine to leave it here for now, thanks for the PR!

**Pull request merged**